src/test/java/com/hocs/test/glue/NavigationStepDefs.java

```diff
@@ -192,7 +203,7 @@ public void iAmTakenToThePage(String pageName) {
         System.out.println("I have been taken to " + pageName);
     }
 
-    private void navigateToHocs() {
+    public void navigateToHocs() {
         String env = System.getProperty("environment");
```

Comment:
What meant this needed to be public?

Reply:
Not sure when that changed but it's been changed to Private